docs/src/reference/cli/changelog.rst

```diff
@@ -1,7 +1,7 @@
 CLI Changelog
 =============
 
-1.0.6 (UNRELEASED)
+1.0.6 (1.0.6rc1)
```

Comment:
These should be dates - sorry if I misspoke during our convo. Something like: `1.0.6rc1 (2021-02-16)`

Reply:
ah ok that makes more sense.  One question, if we have to publish an `rc2`, or when we do the final release, will we be editing this line or will we add a new entry to the changelog?

Reply:
I would say new entry/section

Reply:
ok, and would we copy the lines out of the `rc` releases or would the final release entry just be empty?

Reply:
ah v good question! Thinking about this a little bit more, I think we should change the header rather than the add a new entry section (and of course add any bullets of user facing changes between last rc and this rc/GA release).